### PR TITLE
black: list / dict / tuple formatting

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Part of #99

### CODE REVIEW

**Review by kiorky — COMMENTED** [review]
for this one, im unsure if we could keep as-is all the definitions as it is more compact and easier to read even if this part is not black'ed.

File: src/croniter/croniter.py [hunk 1]
```diff
@@ -100,9 +120,30 @@ def is_32bit():
...
+)
+YEAR_FIELDS = (
+    MINUTE_FIELD,
+    HOUR_FIELD,
+    DAY_FIELD,
+    MONTH_FIELD,
+    DOW_FIELD,
+    SECOND_FIELD,
+    YEAR_FIELD,
+)
```
Comment (kiorky) [inline src/croniter/croniter.py]: i mean up to that part of the code, below, changes are mergeable without any comment.

Comment (evanpurkhiser) [reply src/croniter/croniter.py]: I think if we remove the [magic trailing commma](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma) the items that are not longer than 80 characters will not break into multi lines.

Let me try that.

Comment (evanpurkhiser) [reply src/croniter/croniter.py]: let me pull these out of this PR real quick though so we can merge the rest.

**Comment by evanpurkhiser** [comment]
Will open a new PR to figure out the "alphas"

**Comment by kiorky** [comment]
> Will open a new PR to figure out the "alphas"

`black --line-length 120` should solve this.

and Yes, of course, i will merge the non commentables ones, Thx again for the contribution !

**Comment by evanpurkhiser** [comment]
I can set a longer line length, but unfortunately it may cause a bunch of other reformatting now :(

How do you feel about this? https://github.com/kiorky/croniter/pull/121

**Pull request merged**